### PR TITLE
Hotfix/flash messages

### PR DESCRIPTION
**Pull request by tehreemtalat7**

### CODE REVIEW

File: app/controllers/employees_controller.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class EmployeesController < ApplicationController
+
```
Comment (mabdullahkhalil) [inline app/controllers/employees_controller.rb]: extra line

File: app/controllers/tasks_controller.rb [hunk 2]
```diff
@@ -1,45 +1,63 @@
...
 
   def update
     if !params[:assignable_employee_id].empty? && !params[:assignable_team_id].empty?
-      render :edit, notice: "Make sure you delete employee field or team field"
+      flash[:danger] = 'Make sure you delete employee field or team field'
+      render :edit
     else
-      params[:task][:status] = "new" if params[:task][:assignable_id].empty?
-      params[:task][:status] = "assigned" if !params[:task][:assignable_id].empty?
+      params[:task][:status] = 'new' if params[:task][:assignable_id].empty?
```
Comment (mabdullahkhalil) [inline app/controllers/tasks_controller.rb]: hard coded strings

File: app/views/attendances/_attendance_table.html.erb [hunk 3]
```diff
@@ -14,6 +14,7 @@
   </thead>
 
   <tbody>
+    <% if @attendances_list.length > 0 %>
```
Comment (mabdullahkhalil) [inline app/views/attendances/_attendance_table.html.erb]: if @attendances_list.any?

File: app/controllers/tasks_controller.rb (hunk 2 above)
Comment (tehreemtalat7) [reply app/controllers/tasks_controller.rb]: This is already solved by abdullah in his branch, it'll be updated after merging

**Pull request merged**